packages/nimble-blazor/NimbleBlazor/NimbleBlazor.csproj

```
@@ -50,10 +51,11 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Include="package.json" />
     <None Include="packages.lock.json" />
   </ItemGroup>
 
   <Target Name="CopyNimbleResources" BeforeTargets="BeforeBuild">
-    <Exec Command="npm run copy-resources" />
+    <Exec Command="cd ../ &amp;&amp; npm run copy-resources:nimble" />
```

There is a [WorkingDirectory option](https://learn.microsoft.com/en-us/visualstudio/msbuild/exec-task?view=vs-2022#parameters), something like

```suggestion
    <Exec Command="npm run copy-resources:nimble" WorkingDirectory="../" />
```